opm/test_util/EclFilesComparator.cpp

```diff
@@ -108,15 +108,15 @@ void ECLFilesComparator::deviationsForCell(double val1, double val2, const std::
         std::cout << "(value1, value2) = (" << val1 << ", " << val2 << ")\n\n";
     }
     if (val1 < 0) {
-        if (abs(val1) > absTolerance) {
+        if (std::abs(val1) > absTolerance) {
```

Comment:
I'm 100% with @akva2 on this on - unless you specifically want ADL because you're already in a templated function that can take a custom abs for your custom number type, use explicit `std::abs`. I usually go so far as to *never* import the `std` namespace into the global namespace and always qualify my `std` or third-party libraries.